decidim-core/app/models/decidim/user.rb

```diff
@@ -120,7 +120,7 @@ def unread_conversations
     def self.find_for_authentication(warden_conditions)
       organization = warden_conditions.dig(:env, "decidim.current_organization")
       find_by(
-        email: warden_conditions[:email],
+        email: warden_conditions[:email].downcase,
```

Comment:
I'm not sure if it can happen, but maybe:

```suggestion
        email: warden_conditions[:email].to_s.downcase,
```

To avoid a possible raise.